custom/icds_reports/const.py

```diff
@@ -154,3 +155,5 @@ class MapColors(object):
 PREGNANT_WOMEN_ENROLLED_FOR_ANGANWADI_SERVICES = 'Pregnant women enrolled for Anganwadi Services'
 LACTATING_WOMEN_ENROLLED_FOR_ANGANWADI_SERVICES = 'Lactating women enrolled for Anganwadi Services'
 ADOLESCENT_GIRLS_ENROLLED_FOR_ANGANWADI_SERVICES = 'Adolescent girls enrolled for Anganwadi Services'
+ADOLESCENT_GIRLS_ENROLLED_FOR_ANGANWADI_SERVICES_V2 = 'Out of school Adolescent girls(11-14 years)'
```

Comment:
I see that all constants are named exactly like the string. Can/Should this be kept the same to maintain consistency?

Reply:
hey, Not sure I follow.

Reply:
I see what you are saying. Yeah, we can do that.